patroni/__init__.py

```diff
@@ -23,6 +23,9 @@ def __init__(self):
         self.setup_signal_handlers()
 
         self.version = __version__
+        if len(sys.argv) >= 2 and sys.argv[1] and sys.argv[1] == "--version":
```

Comment:
IMO this is too hacky.
Such stuff should be done in the `main()` or in the `patroni_main()`.
Also, the check `and sys.argv[1]` seems to be unnecessary.

Reply:
Yeah, I was thinking about the same, there is no single entry point for parsing the arguments, I copied this part from Config class, the better solution I was thinking about to rewrite the whole config/environment variables/arguments completely, but unless I have certain goals it seems too big task to solve only version output.